modules/relay/src/main/RelayNotifier.scala

```diff
@@ -0,0 +1,29 @@
+package lila.relay
+
+import chess.format.pgn.{ Tag, Tags }
+import chess.format.UciPath
+import lila.common.licon
+import lila.notify.{ BroadcastRound }
+import lila.socket.Socket.Sri
+import lila.study.*
+import lila.tree.Branch
+
+final private class RelayNotifier(
+    notifyApi: lila.notify.NotifyApi,
+    tourRepo: RelayTourRepo
+)(using Executor):
+
+  def roundBegin(rt: RelayRound.WithTour): Funit =
+    tourRepo
+      .subscribers(rt.tour.id)
+      .flatMap { subscribers =>
+        notifyApi.notifyMany(
+          subscribers,
+          BroadcastRound(
+            s"/broadcast/${rt.tour.slug}/${rt.round.slug}/${rt.round.id}",
```

Comment:
path functions available at https://github.com/lichess-org/lila/blob/master/modules/relay/src/main/RelayRound.scala#L132-L134

Reply:
`:thanks:` (we need tom hanks emoji in github)